src/clj/puppetlabs/puppetserver/jruby_request.clj

```diff
@@ -67,10 +67,10 @@
      (handler (assoc request :jruby-instance jruby-instance)))))
 
 (defn get-environment-from-request
-  "Gets the environment from a request."
+  "Gets the environment from the URL or query string of a request."
   [req]
-  (-> req
-      (get-in [:params "environment"])))
+  (or (get-in req [:params :environment])
+      (get-in req [:params "environment"])))
```

Comment:
Might be worth adding a comment here explaining the difference between the two.

@camlow325 @haus do you have any concerns about this in terms of the other endpoints?  Seems like it should be fine?